.github/workflows/notify_release.yml

```diff
@@ -0,0 +1,31 @@
+# A workflow runs when a release is published, dispatches a new event to the vercel/turbo
+# to notify its release. Turbopack, and other integration workflow will subscribe to this event.
+name: Notify new Next.js release
+on:
+  release:
+    types: [published]
+
+jobs:
+  notify:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v6
```

Comment:
Can this just go in `build_test_deploy` after the release finishes similar to release stats/e2e deploy tests https://github.com/vercel/next.js/blob/57d0e77f78d229929828b167691eb861e5292f14/.github/workflows/build_test_deploy.yml#L837

Reply:
I think so, as long as it is guaranteed event for the release. The reason I choose  workflow event (`release`) is it automatically carries tag / release version through `context.ref`. If we move this over, is there pre-defined ref I can use or I need to look for it by myself?

Reply:
Hmm if the ref is needed for checking out the repo and we're not relying on the npm version being published should be alright here instead

Reply:
Yes, currently it relies on direct checkout - since our purpose is to run integration test which are not available in the published pkg, we should need to clone / checkout repo anyway